# NGS-2022spring

Possibly subsample

Remove adapters and quality control (AdapterRemoval, Trimmomatic, cutadapt, fastp)

Align reads to reference genome, fastq → bam (BWA mem)

Mark duplicates (Picard)

Get variants from alignment, bam → vcf (GATK HaplotypeCaller)

Inspect mutations in the pncA gene including a potential deletion

Construct a phylogenetic tree, vcf → fasta → phylip(phyml)

Visualize the phylogenetic tree (iTOL)

Do PCA analysis to check possible clusters (R)

Predict drug resistance (TB-profiler)

Google slide
https://docs.google.com/presentation/d/1KUhntjp3vhhEze0aiLoQEDJpCLuf9aIU8Tj2VUcvwIA/edit#slide=id.p

Overleaf Poster 
https://www.overleaf.com/2728134937brjsxdqhwxvh

